### PR TITLE
22w11a & 12a datafixer mappings

### PR DESCRIPTION
**Pull request by IotaBread**

### CODE REVIEW

File: mappings/net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/unmapped/C_frjcqtfh net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix
```
Comment (OroArmor) [inline mappings/net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix.mapping]: Delta? Or delete?

Comment (NoahvdAa) [inline mappings/net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix.mapping]:
```suggestion
CLASS net/minecraft/unmapped/C_frjcqtfh net/minecraft/datafixer/fix/ChunkDeleteIgnoredLightDataFix
```

Comment (IotaBread) [reply mappings/net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix.mapping]: Delete

Comment (IotaBread) [reply mappings/net/minecraft/datafixer/fix/ChunkDelteIgnoredLightDataFix.mapping]: Fixed

**Comment by NoahvdAa** [comment]
Don't forget to rename the file aswell

**Comment by IotaBread** [comment]
I forgot, fixed

**Review by NoahvdAa — APPROVED** [review]

**Review by OroArmor — APPROVED** [review]

**Review by NoComment1105 — APPROVED** [review]

**Pull request merged**